keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterSpec.kt

```diff
@@ -51,23 +55,45 @@ data class TitusClusterSpec(
   private val _defaults: TitusServerGroupSpec,
   @JsonInclude(JsonInclude.Include.NON_EMPTY)
   val overrides: Map<String, TitusServerGroupSpec> = emptyMap(),
-  @JsonIgnore
-  override val deliveryArtifact: DeliveryArtifact? = null,
-  @JsonIgnore
+  // Custom backing field for artifactName, used by resolvers
+  private val _artifactName: String? = null,
```

Comment:
This allows resolvers to write the property while still giving us the ability to provide a custom getter below which attempts to find the artifact info _before_ resolvers have run.